# If $ac_cv_func_dlopen is not 'yes' a bit earlier, SHOBJ_LIBS remains
# undefiled. Later during 'make install' stage, examples/loadables/Makefile
# calls install-$(SHOBJ_STATUS), and since the target 'install-' is undefined,
# make prints error.
#
# Reported to bug-bash@gnu.org

--- configure.ac	2016-12-06 05:07:31.808614344 +0000
+++ configure.ac	2016-12-06 05:07:26.791244255 +0000
@@ -1151,6 +1151,8 @@ then
 	AC_SUBST(SHOBJ_LIBS)
 	AC_SUBST(SHOBJ_STATUS)
 	AC_MSG_RESULT($SHOBJ_STATUS)
+else
+	SHOBJ_STATUS=unsupported
 fi
 
 # try to create a directory tree if the source is elsewhere
